packages/runtime/src/runtimes/react/controls/rich-text-v2/EditableTextV2/editable-text-v2.tsx

```diff
@@ -189,7 +189,7 @@ export function EditableTextV2({ text, definition, control }: Props) {
 
   const handleClick = useCallback(
     (e: MouseEvent) => {
-      if (editMode === BuilderEditMode.CONTENT){
+      if (editMode === BuilderEditMode.CONTENT) {
```

Comment:
Ran prettier on the `src/runtimes/react` folder and caught some formatting issues here and there